python/lsst/daf/butler/registry/datasets/byDimensions/_manager.py

```diff
@@ -41,7 +48,7 @@
 
 
 # This has to be updated on every schema change
-_VERSION = VersionTuple(0, 1, 0)
+_VERSION = VersionTuple(2, 0, 0)
```

Comment:
Skipping 1.x.y?

Reply:
There was another commit on the same branch that went to 1.0.0.  I figure it's probably better to increment versions by commit rather than by branch when possible, but I could be persuaded otherwise.

Reply:
Well, my mental model is that we are effectively squashing commits in the same PR, to me it means that version 1.0.0 will never be officially released and there is no point providing migration from 0.1.0 to 1.0.0 as there is no software that can work with that version. I'm OK with gaps in versions, it may just look odd to people who need to look at version numbers.